packages/adapter-cloudflare/index.js

```diff
@@ -116,6 +116,13 @@ function get_routes_json(builder, assets, { include = ['/*'], exclude = ['<all>'
 
 			return rule;
 		});
+    
+    if (!exclude.filter(rule => rule === "<build>" || rule === "/_app/*").length) {
```

Comment:
> would probably need to check for `<all>` as well

That wouldn't work because `<all>` gets mapped to `<build>`, `<files>` and `<prerendered>`.

This did make me realize that I made a mistake in checking after the mapping, and now the check happens before.
Checking after would mean that `<build>` would get converted to the app dir so there'd be no reason to check `<build>` in the first place.

And I was also checking for a hard-coded app directory which would break the check if the default app directory changes.